exercises/practice/pangram/pangram_test.gd

```diff
@@ -0,0 +1,57 @@
+func test_empty_sentence(solution_script):
+        var input = ""
```

Comment:
Here I think you are using spaces instead of tabs, could you please switch to tabs instead? (they are the official standard in Godot editor)